tests/datadriven/test_variants.py

```diff
@@ -192,6 +193,14 @@ def _verifyVariantCalls():
                     self.assertEqual(str(alt1), str(alt2))
             else:
                 self.assertEqual(gaVariant.alternate_bases, alt)
+            if not gaVariant.filters_applied:
```

Comment:
There are variants with no filters applied (marked with '.') in tests/data/datasets/dataset1/variants/WASH7P_annotation/WASH7P_annotation.vcf.gz.